packages/repository/test/fixtures/repositories/order.repository.ts

```diff
@@ -12,14 +17,14 @@ export class OrderRepository extends DefaultCrudRepository<
   Order,
   typeof Order.prototype.id
 > {
-  public customer: BelongsToFactory<Customer, Order>;
+  public customer: BelongsToFactory<Customer, typeof Order.prototype.id>;
```

Comment:
If you rename `createBelongsToFactory` per my comment above, then please rename `BelongsToFactory` type accordingly too.